CHANGELOG.md

```diff
@@ -6,6 +6,11 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
 
+### Changed 
```

Comment:
There is already a `Changed` section, please move your entry down.
Please select `Squash and merge` when merge your PR.